mycoffee/functions.py

```diff
@@ -135,7 +136,9 @@ def save_result_text(params: Dict[str, Union[str, int, float, dict]],
     if not ignore_warnings:
         warnings_list = params["warnings"]
         if len(warnings_list) > 0:
-            result = result + "\n\n" + "\n".join(warnings_list)
+            result += "\n\n"
+            for message in warnings_list:
+                result += "[Warning] " + message + "\n"
```

Comment:
Also the same for this.

Reply:
I believe it's not a significant issue, and it may also improve readability.